src/dlr_tvm.cc

```diff
@@ -214,3 +223,32 @@ void TVMModel::UseCPUAffinity(bool use) {
     LOG(INFO) << "CPU Affinity is disabled";
   }
 }
+
+bool TVMModel::HasMetadata() const {
+  return this->metadata != nullptr;
+}
+
+const char* TVMModel::GetOutputName(const int index) const {
+  std::string name_str = this->metadata["Model"]["Outputs"][index]["name"];
```

Comment:
Lets use `vector<string> output_names_` which is filled with output names from `metadata` in the constructor.

Reply:
I don't think its necessary. Why do we need to do that?

Reply:
Adding unecessary indirection to the actual data.

Reply:
I agree we shouldn't need to create `output_names_`.

However this function should return
```
return this->metadata["Model"]["Outputs"][index]["name"].c_str();
```

Since `name_str.c_str()` will be invalid when `name_str` goes out of scope (when this function ends).

Reply:
👍